src/core/credrank/markovProcessGraph.test.js

```diff
@@ -380,6 +393,12 @@ describe("core/credrank/markovProcessGraph", () => {
     });
   });
 
+  describe("toMarkovChain", () => {
+    it("does not violate its own invariant checker", () => {
+      markovProcessGraph().toMarkovChain();
+    });
+  });
```

Comment:
Added this because this failed for a bad implementation, and having it is helpful for debugging.